daemon/smartcard.c

```diff
@@ -467,9 +486,81 @@ smartcard_cb_start_container(int fd, unsigned events, event_io_t *io, void *data
 	}
 }
 
+static void
+smartcard_cb_stop_container(int fd, unsigned events, event_io_t *io, void *data)
+{
+	smartcard_startdata_t *startdata = data;
+	int resp_fd = control_get_client_sock(startdata->control);
+
+	IF_TRUE_GOTO(events & EVENT_IO_EXCEPT, exit);
+
+	IF_FALSE_RETURN(events & EVENT_IO_READ);
+
+	// use protobuf for communication with scd
+	TokenToDaemon *msg =
+		(TokenToDaemon *)protobuf_recv_message(fd, &token_to_daemon__descriptor);
+
+	if (!msg) {
+		ERROR("Failed to receive message although EVENT_IO_READ was set. Aborting container stop.");
+		goto exit;
+	}
+
+	switch (msg->code) {
+	case TOKEN_TO_DAEMON__CODE__LOCK_FAILED: {
+		WARN("Locking the token failed.");
+		control_send_message(CONTROL_RESPONSE_CONTAINER_START_LOCK_FAILED, resp_fd);
```

Comment:
wrong response codes are used here and in other case statements! START -> STOP

Reply:
Done (+fixed some more inconsistencies)